clusters-rke2/check_etcd_perf.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+### This script will loop through all generated kubeconfig files in the "./files" directory based from wherever the script was invoked.
+### It will look for kubeconfig files matching the pattern of ".workspace-XXXXX_kube_config", where each X = a number. It will then execute the etcdctl check perf command.
```

Comment:
Where are the .workspace kube configs being generated?

Reply:
This is an artifact from the RKE1 modules that I hadn't finished implementing for RKE2 yet. I will update the RKE2 module to create these.

Reply:
@floatingman This should be fixed